internal/dataexchange/ffdx/ffdx.go

```diff
@@ -366,6 +370,11 @@ func (h *FFDX) eventLoop() {
 				} else {
 					err = h.callbacks.BLOBReceived(msg.Sender, *hash, msg.Size, msg.Path)
 				}
+			case blobAcknowledged:
+				err = h.callbacks.TransferResult(msg.RequestID, fftypes.OpStatusSucceeded, fftypes.TransportStatusUpdate{
```

Comment:
I don't think we have any handling for `blob` generation of a `manifest` - so the change to fully support this might be a bit larger

Reply:
Added the logic to store the blob hash as the operation input. This information is later used to check the data coming back from the acknowledgement, similar to how manifests are checked for batch messages.